### DISCUSSION TITLE
Can't load config, can't get terminal window size

### DISCUSSION THREAD

**Discussion by allreddv** [3rd/image.nvim#228] [Q&A] [answered]
Hello,   I am all of a sudden sometimes getting an error message on opening nvim that says can't run config for image.nvim because it can't determine terminal window size.  Any idea on why this is popping up,  it is also only about 25% of the time.  I have not been able to connect a previous action to when it happens so I am not sure what might be the difference for it triggering or not.   Any ideas?

**Comment by 3rd** [comment]
This is weird, are you on Kitty?

**Reply by allreddv** [reply]
no, wezterm. Sounds like a random issue and not one you are not familiar with so  I will grab the full error message when It happens again and report back.  Thanks.

**Reply by allreddv** [reply]
Okay just got the error again.  Here it is. 

Failed to run `config` for image.nvim

...ocal/share/nvim/lazy/image.nvim/lua/image/utils/term.lua:34: Failed to get terminal size

# stacktrace:
  - /image.nvim/lua/image/utils/term.lua:34 _in_ **update_size**
  - /image.nvim/lua/image/utils/term.lua:46
  - /image.nvim/lua/image/utils/init.lua:8
  - /image.nvim/lua/image/init.lua:1
  - ~/.config/nvim/lua/dvavim/plugins/image-nvim.lua:45 _in_ **config**
  - ~/.config/nvim/lua/dvavim/lazy.lua:15
  - ~/.config/nvim/init.lua:2


any ideas?

**Reply by 3rd** [reply, marked as answer]
probably a Wezterm issue, they've had them in the past around term size

**Reply by allreddv** [reply]
Ok thanks, i will see if i can find anything out from discussions on wezterm.  I can also try running through tmux with wezterm 
term =  "xterm-kitty"   I am having some issues seeing images correctly in .md files and saw a YT video explaining to do this and it fixes it,  maybe it will resolve this error issue also.  In case anyone else reads this and has similar issues the video is from channel linkarzu
